### DISCUSSION TITLE
Adonis 5 heroku error

### DISCUSSION THREAD

**Discussion by jhovannycanas** [adonisjs/core#2617] [Help] [answered]
when i deploy the app in heroku i get the following error, i have followed the previous threads settings of the command in the package.

Exception: E_MISSING_ENV_VALUE: Missing environment variable "HOST"

**Comment by RomainLanz** [comment, marked as answer]
Hey @jhovannycanas! 👋 

Have you follow our guide?
https://docs.adonisjs.com/cookbooks/deploy-to-heroku

**Reply by jhovannycanas** [reply]
hi @RomainLanz  yes...and any related guides found

**Reply by jhovannycanas** [reply]
I solved it, it is necessary to place the host variable in the configuration of variables in heroku

![image](https://user-images.githubusercontent.com/19630565/118363209-e6458f80-b558-11eb-8421-c2be9d93bd7d.png)
